app/controllers/katello/api/v2/content_views_controller.rb

```diff
@@ -79,13 +79,25 @@ def update
     param :description, String, :desc => N_("Description for the new published content view version")
     param :force_yum_metadata_regeneration, :bool, :desc => N_("Force yum metadata regeneration on the repositories " \
                                                            "in the content view version")
-    param :major, :number, :desc => N_("Override the major version number"), :required => false
-    param :minor, :number, :desc => N_("Override the minor version number"), :required => false
+    param :major, :number, :desc => N_("Override the major version number")
+    param :minor, :number, :desc => N_("Override the minor version number")
+
+    param :repos_units, Hash, :desc => N_("Specify the list of units in each repo") do
```

Comment:
Should this be required false?

Reply:
Had to change this to :required => false so hammer would be happy and let me publish, after talking to @ehelms api/webui will not care about this but hammer hard fails with required off.

Reply:
@chris1984 ah interesting, thx